src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rentCar = 40;
+  let sumRent = 0;
+
+  switch (true) {
+    case (days >= 3 && days <= 6):
+      sumRent = (days * rentCar) - 20;
+      break;
+    case (days >= 7):
+      sumRent = (days * rentCar) - 50;
+      break;
+    default:
```

Comment:
We usually use `switch case` when we have a defined count of cases. This task looks better for using several plain `if`s